hadoop-hdds/rocks-native/src/CMakeLists.txt

```diff
@@ -22,8 +22,8 @@
 
 cmake_minimum_required(VERSION 2.8)
 add_definitions(-D_GLIBCXX_USE_CXX11_ABI=0)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
Why no optimization at all? This is different that specifying the instruction sets. Can you look into `add_compile_options(-msse2) ` ? I am not sure no optimization == chipset instructions to be used.

Reply:
I am not really aware of this, but let me check this out and get back on this.